backend/apps/project_health/services.py

```
@@ -93,19 +109,21 @@ def _compute_health_score(data: dict) -> tuple[int, list, list]:
     contributors = data.get("contributor_count", 0)
+
     if contributors > 50:
         score += 10
         green_flags.append(
             f"✅ Large contributor community ({contributors} contributors)."
         )
     elif contributors < 5:
         score -= 5
-        red_flags.append("🟡 Low contributor count – bus-factor risk.")
+        red_flags.append(
+            "🟡 Low contributor count – bus-factor risk."
```

Ruff (0.15.21)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)